llama.cpp

```diff
@@ -691,6 +698,7 @@ struct llama_mmap {
         }
 
         if (prefetch) {
+#if WINVER >= 0x0602
```

Comment:
What part of the below code does not compile on older Windows? It's dynamically loading the symbol.

Reply:
This symbol not available in the headers: PWIN32_MEMORY_RANGE_ENTRY

Reply:
I see. The dynamic bit was so you can compile _for_ Windows 7, but you are compiling _on_ Windows XP. I think you should add a comment like `// Windows 8` to clarify what 0x0602 represents.

Reply:
Addressed